src/components/member-profile/contribution/index.js

```diff
@@ -64,7 +68,16 @@ const ContributionCard = ({
   urlObj,
 }) => {
   const {
-    task: { id, title, startedOn, endsOn, status, purpose, isNoteworthy },
+    task: {
+      id,
+      title,
+      startedOn,
+      endsOn,
+      status,
+      purpose,
+      isNoteworthy,
+      isCollapsed,
+    },
     prList,
```

Comment:
Isnt if am not wrong prList is an array of object also it's on 0th idx where is the zero index destructure done?